### DISCUSSION TITLE
Ajuda com instalação - GD

### DISCUSSION THREAD

**Discussion by Peffmb** [RamonSilva20/mapos#1907] [Q&A]
Não consigo instalar o Mapos, fiz mesmo processo anteriormente mas agora não consigo prosseguir com a configuração. segue print do que não funciona: https://prnt.sc/gtWBfPk4V2wL

**Comment by michaellrz** [comment]
GD é uma extensão do PHP, a forma como você vai instalar vai depende do ambiente q está usando… Linux… Windows…

**Comment by Peffmb** [comment]
Instalei windows, composer padrão, vou ver como instala padrão.

Em dom., 28 de ago. de 2022 00:05, Michael Rezende ***@***.***>
escreveu:

**Comment by SuperPorte** [comment]
Testa isso:

https://github.com/RamonSilva20/mapos/discussions/1929#discussioncomment-3916078
